src/algebra/category/Module/change_of_rings.lean

```diff
@@ -13,6 +14,18 @@ import algebra.category.Module.basic
 * `category_theory.Module.restrict_scalars`: given rings `R, S` and a ring homomorphism `R ⟶ S`,
   then `restrict_scalars : Module S ⥤ Module R` is defined by `M ↦ M` where `M : S-module` is seen
   as `R-module` by `r • m := f r • m` and `S`-linear map `l : M ⟶ M'` is `R`-linear as well.
+
+* `category_theory.Module.extend_scalars`: given **commutative** rings `R, S` and ring homomorphism
+  `f : R ⟶ S`, then `extend_scalars : Module R ⥤ Module S` is defined by `M ↦ S ⨂ M` where the
+  module structure is defined by `s • (s' ⊗ m) := (s * s') ⊗ m` and `R`-linear map `l : M ⟶ M'`
+  is sent to `S`-linear map `s ⊗ m ↦ s ⊗ l m : S ⨂ M ⟶ S ⨂ M'`.
+
+## List of notations
+Let `R, S` be rings and `f : R →+* S`
+* if `M` is an `R`-module, then notation `S ⨂[R, f] M` means the tensor product `S ⨂ M` where `S`
+  is considered as an `R`-module via restriction of scalars.
```

Comment:
Does this notation still exist?

Reply:
Indeed, it no longer exists and has been deleted.